content/hi/docs/reference/_index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: संदर्भ
+approvers:
+- chenopis
+linkTitle: "संदर्भ"
+main_menu: true
+weight: 70
+content_type: concept
+no_list: true
+---
+
+
+<!-- overview -->
+
+कुबेरनेट्स प्रलेखन के इस खंड में संदर्भ हैं।
+
+
+
+<!-- body -->
+
+## API संदर्भ
+
+* [शब्दकोष](/docs/reference/glossary/) - कुबेरनेट्स शब्दावली की एक व्यापक, मानकीकृत सूची।
+* [कुबेरनेट्स API संदर्भ](/docs/reference/kubernetes-api/)
+* [कुबेरनेट्स के लिए एक-पृष्ठ API संदर्भ {{< param "version" >}}](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/)
+* [कुबेरनेट्स API का उपयोग करना](/docs/reference/using-api/) - कुबेरनेट्स के लिए API का अवलोकन।
+* [API अभिगम नियंत्रण](/docs/reference/access-authn-authz/) - कुबेरनेट्स API एक्सेस को कैसे नियंत्रित करता है, इस पर विवरण।
+* [जाने-माने लेबल, टिप्पणी और दाग](/docs/reference/labels-annotations-taints/)
+
+## आधिकारिक तौर पर समर्थित ग्राहक पुस्तकालय
+
+कुबेरनेट्स API को प्रोग्रामिंग भाषा से कॉल करने के लिए, आप उपयोग कर सकते हैं
+[client libraries](/docs/reference/using-api/client-libraries/). आधिकारिक तौर पर समर्थित
+client libraries:
+
+- [कुबेरनेट्स Go client library](https://github.com/kubernetes/client-go/)
+- [कुबेरनेट्स Python client library](https://github.com/kubernetes-client/python)
+- [कुबेरनेट्स Java client library](https://github.com/kubernetes-client/java)
+- [कुबेरनेट्स JavaScript client library](https://github.com/kubernetes-client/javascript)
+- [कुबेरनेट्स C# client library](https://github.com/kubernetes-client/csharp)
+- [कुबेरनेट्स Haskell client library](https://github.com/kubernetes-client/haskell)
+
+## CLI
+
+* [kubectl](/docs/reference/kubectl/overview/) - कमांड चलाने और कुबेरनेट्स क्लस्टर के प्रबंधन के लिए मुख्य CLI उपकरण।
+    * [JSONPath](/docs/reference/kubectl/jsonpath/) - kubectl के साथ प्रयोग करने के लिए 
+वाक्य - विन्यास मार्गदर्शक [JSONPath expressions](https://goessner.net/articles/JsonPath/).
```

Comment:
I feel like we can use `वाक्य-रचना` for syntax as it is more simpler to understand and use. Thoughts?

Reply:
Syntax is a term used in context of programming language (JSONPath here). I think we should go with transliteration for syntax to avoid confusions